recipes/mlrose-ky/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mlrose-ky" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/d9/12/db9f576553caa2df5b435a8e941ff29426c199e5aaee28b6967a97d4682a/mlrose_ky-1.1.6.tar.gz
+  sha256: 50a7e2fec9ca6c64418809185acf96aa220d4c62ae8d67519a30bd281dcc7758
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - setuptools
+  run:
+    - python >=3.10
+    - joblib
+    - matplotlib-base
+    - networkx
+    - numpy
+    - pandas
+    - scikit-learn
+    - scipy
+
+test:
```

Comment:
Can you add a `pip check`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L64-L67

Reply:
Done. [482b792](https://github.com/conda-forge/staged-recipes/pull/27859/commits/482b792a37ea98244462fc3a221159045e216f93)